app.go

```diff
@@ -84,8 +84,8 @@ type ErrorHandler = func(*Ctx, error) error
 
 // Error represents an error that occurred while handling a request.
 type Error struct {
-	Code    int    `json:"code"`
-	Message string `json:"message"`
+	Code    int         `json:"code"`
```

Comment:
nit: we can get rid of extra spaces

Reply:
interface{} is longer than string so editor has aligned automatically it.